specification/storage/resource-manager/Microsoft.Storage/stable/2025-01-01/storage.json

```diff
@@ -2665,6 +2686,27 @@
           },
           "description": "The set of locations that the SKU is available. This will be supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.)."
         },
+        "locationInfo": {
```

Comment:
I believe SRP team has some concern on adding "locationInfo" into swagger before. 
>we cannot add the "locationInfo" to swagger until after we are able to ‘turn off’ the zone list that’s currently being returned

Please confirm SRP team agree to add "locationInfo" to swagger.